input/janus_mixed.cfg

```diff
@@ -30,15 +30,15 @@ iter_max                    = 9000
 log_level                   = DEBUG
 
 # Output subdirectory name (relative to output folder)
-dir_output                  = janus_mixed
+dir_output                  = jans_mixed_hot
 
 # Choose times
 time_star                   = 100.0e6  	# yr, time since star formation
 time_planet                 = 0.        # yr, time since MO start
 time_target                 = 4.567e+9  # yr, target time for MO evolution
 
 # SOCRATES spectral file to use (relative to JANUS folder)
-spectral_file               = AGNI/res/spectral_files/nogit/Dayspring/48/Dayspring.sf
+spectral_file               = JANUS/spectral_files/Dayspring/256/Dayspring.sf
 
```

Comment:
Seeing this line, I am wondering if you plan to mimic what we did in Janus in AGNI, i.e. downloading/storing spectral data files in the fwl_data repository?

Reply:
I am not sure what the best thing is to do with that. The current disk size of AGNI with all the default files is now about 600 MB, which is too large.

I think storing all of the spectral files in a single place makes the most sense, so maybe writing a simple wrapper for the OSF repo in AGNI would be a good idea.

Reply:
On the Proteus side, it makes sense to have the data stored once in a place that both AGNI and Janus share access.